### PR TITLE
[terra-tabs]  Fix for the focus traverse within the tab content.

### PR DESCRIPTION
**Pull request by ST063655**
### Summary

**What was changed:**

1. When a tab is activated, the focus is set to the first focusable element within the tab content.
2. When a tab content for a tab pane is not passed, the tab index is set to -1.

**Why it was changed:**

There was an issue with the keyboard navigation behavior in Terra-tabs. when a tab is activated, the subsequent focusable element becomes the tab-panel div, rather than the focusable element within the tab content. 
**expected behavior:** The focus should move to the first focusable element within the newly activated tab content.

### Testing

This change was tested using:

- [x] WDIO
- [x] Jest
- [ ] Visual testing (please attach a screenshot or recording)
- [ ] Other (please describe below)
- [ ] No tests are needed

### Reviews

In addition to engineering reviews, this PR needs:

- [ ] UX review
- [ ] Accessibility review
- [x] Functional review

### Additional Details

**This PR resolves:**

UXPLATFORM-9598 

---

Thank you for contributing to Terra.
@cerner/terra

### CODE REVIEW

**Comment by saket2403** [comment]
Can you add a wdio test for these changes?

File: package.json [hunk 1]
```diff
@@ -128,6 +128,8 @@
     "terra-application": "^1.48.0",
     "terra-disclosure-manager": "^4.9.0",
     "terra-enzyme-intl": "^3.0.0",
+    "terra-form-field": "^4.27.0",
+    "terra-form-fieldset": "^2.74.0",
```
Comment (supreethmr) [inline package.json:132]: why do we need these dependencies ..??

Comment (ST063655) [reply package.json:132]: Added the dependencies as part of terra-framework-docs package.json.

File: packages/terra-tabs/CHANGELOG.md [hunk 2]
```diff
@@ -2,6 +2,9 @@
 
 ## Unreleased
 
+* Changed
+  * Fix for the focus traverse within the tab content .
```
Comment (supreethmr) [inline packages/terra-tabs/CHANGELOG.md:6]:
```suggestion
* Added
  * Added prop to support the focus traverse within the tab content.
```

**Review by supreethmr — APPROVED** [review]
+1 with change log update

**Review by saket2403 — APPROVED** [review]

**Pull request merged**